website/docs/installation/weave-gitops-enterprise/index.mdx

```diff
@@ -243,6 +244,44 @@ kubectl create secret generic git-provider-credentials --namespace=flux-system \
   --from-literal="GIT_HOST_TYPES=git.example.com=gitlab"
 ```
 
+</TabItem>
+<TabItem value="bitbucket-server" label="BitBucket Server">
+
+Create a new [incoming application link](https://confluence.atlassian.com/bitbucketserver/configure-an-incoming-link-1108483657.html) from 
+the BitBucket administration dashboard. You will be asked to enter a unique name and the redirect URL for the external application. The redirect URL
+should be set to `<WGE dashboard URL>/oauth/bitbucketserver`. You will also need to select permissions for the application. The minimum set of
+permissions needed for WGE to create pull requests on behalf of users is `Repositories - Write`. An example of configuring these settings is shown below.
+
+<figure>
+
+<img src={oauthBitbucket} width="500"/>
+
+<figcaption>Configuring a new incoming application link</figcaption>
+</figure>
+
+
+Save your application and take note of the **Client ID** and **Client Secret** and save
+them into the `git-provider-credentials` secret along with:
+
+- `GIT_HOST_TYPES` to tell WGE that the host is bitbucket-server
+- `BITBUCKET_SERVER_HOSTNAME` where the OAuth app is hosted
+
+**Replace values** in this snippet and run:
+
+```bash
+kubectl create secret generic git-provider-credentials --namespace=flux-system \
```

Comment:
[only a note]
I think it's worth to mention to "create **or update existing secret**".
A cluster can have a `git-provider-credentials` secret already and a user may think "it's already there, I'll just delete the old one" and that can be a painful step.

I don't think it's in the scope of this PR, but long term, maybe we should update this part of the docs to split it into a "generic" section and a provider specific one. In this form, if someone reads the whole page, they will get it, but if someone has a cluster already somewhere, they try to add a new provider, they will just scroll down to this part and if they were not experts, that can cause pain.